py/runai_model_streamer/runai_model_streamer/distributed_streamer/tests/test_partition.py

```
@@ -1,95 +1,86 @@
+        """Verifies the correctness of the source maps for partitioned ranges."""
         total_mapped_chunks = 0
         original_chunk_data = {}
-        for req_idx, req in enumerate(original_requests):
-            current_offset = req.offset
-            for chunk_idx, chunk_size in enumerate(req.chunks):
-                if chunk_size > 0:
-                    original_chunk_data[(req_idx, chunk_idx)] = (req.path, current_offset, chunk_size)
-                current_offset += chunk_size
-        
+        for req in original_requests:
+            for chunk_idx, (offset, size) in enumerate(zip(req.offsets, req.sizes)):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)